### PR TITLE
Mileston 5: Blog app - controllers specs

### PR DESCRIPTION
**Pull request by Gardimy**
In for this milestone, I set up tests on the controllers I recently added to my Blog app.

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @Gardimy 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- You have submitted the pull request for a branch that has no commits. Kindly implement the features of this milestone on another branch and commit them so that a reviewer can effectively review the pull request 👍 😎 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Gambit142 — COMMENTED** [review]
Hi @Gardimy 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Kindly implement the changes I suggested earlier. 👍  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Gardimy** [comment]
You make me Confuse.

What does It mean to commit for you???

**Comment by Davidosky007** [comment]
Hi @Gardimy 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't show any file changes, Hence I can not properly review your project , kindly ensure all changes made on this PR shows up on the highlighted section of the screenshot below 👇 
image

* Your PR file changed section should look like this 
image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -1,10 +1,28 @@
...
 
 RSpec.describe 'Posts', type: :request do
+  describe 'GET /users/:user_id/posts' do
+    let(:user) { User.create(name: 'Test User') }
+
+    it 'returns a successful response' do
+      get user_posts_path(user)
+      expect(response).to have_http_status(200)
+    end
+
```
Comment (Davidosky007) [inline spec/requests/posts_spec.rb:11]: * Kindly fix these failing tests 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Gardimy 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

* Well-written PR 👍 
* No linter errors 👍 
* Good looking readme 👍 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**